assets/schema/ensemble_schema.json

```diff
@@ -3336,7 +3336,93 @@
           "properties": {
             "BottomNavBar": {
               "description": "Use the bottom navigation bar",
-              "$ref": "#/$defs/MenuBase"
+              "allOf": [
```

Comment:
You don't need allOf here, you can combine the two objects into one schema

Reply:
I just updated it. Can you review it again?